From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Sat, 1 Oct 2016 16:54:45 +0200
Subject: man: drop myhostname from recommended nsswitch host configuration

resolve includes myhostname functionality, so there is no need to add it again.
---
 man/nss-resolve.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/nss-resolve.xml b/man/nss-resolve.xml
index d9e5645..c5ce19b 100644
--- a/man/nss-resolve.xml
+++ b/man/nss-resolve.xml
@@ -85,7 +85,7 @@
 group:          compat mymachines
 shadow:         compat
 
-hosts:          files mymachines <command>resolve</command> myhostname
+hosts:          files mymachines <command>resolve</command>
 networks:       files
 
 protocols:      db files
